src/components/LinkCard/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from '@trussworks/react-uswds';
+
+import './index.scss';
+
+type LinkCardProps = {
+  children: React.ReactNode;
+  className?: string;
+  link: string;
+  heading: React.ReactNode | string;
+} & JSX.IntrinsicElements['div']; // what does this mean?
```

Comment:
In short, this extends what "intrinsic" props can be passed to this component.

We are able to pass intrinsic attributes to `div`. For example, `aria-label`, `title`, `data-*`, etc. Then we can spread the props onto the parent `div`.

In order for it to be useful, you'd need to destructure the remaining props, then spread them on to the parent `div`.

Reply:
Aah, thank you.  That makes some sense.  So is the `&` like a `&&`, as in "these named props AND all these intrinsic ones"?

Reply:
I added in `...props`- let me know if I'm doing what you explained.  Does it make sense to pass them if I'm not using them? (I think I got an error when I didn't have the `& JSX.IntrinsicElements['div'];`